docs/content/interacting/consistency.mdx

```diff
@@ -51,19 +51,22 @@ if (date_modified + cache_time_to_live_period > Date.now()) {
 
 ## Cache expiration
 
-<ProductName format={ProductNameFormat.ShortForm}/> cache is disabled by default. When not enabled, all queries will have strong consistency regardless of the consistency mode specified.
+<ProductName format={ProductNameFormat.ShortForm}/> caching is disabled by default. When caching is disabled, all queries will have strong consistency regardless of the consistency mode specified. When caching is enabled, the cache will be used for queries with `MINIMIZE_LATENCY` consistency mode.
 
 You can use the following parameters to configure <ProductName format={ProductNameFormat.ShortForm}/>'s cache:
 
-| Name                    | Description                                                                     |
-|-------------------------|---------------------------------------------------------------------------------|
-| check-query-cache       | Enables caching (default = false)                                               | 
-| check-query-cache-limit | Configures the number of items that will be kept in the cache (default = 10000) | 
-| check-query-cache-ttl   | Specifies the time that items will be kept in the cache (default = 10s)         |	
+| Name                             | Description                                                                                                                                                                                                                                                                                                                                                                     |
+|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Looks good. Is this a breaking change? e.g. do we need to tell customers to configure caching differently after they update to a specific version?

Reply:
The new caching flags are mentioned in the upcoming release notes: https://github.com/openfga/openfga/blob/main/CHANGELOG.md#unreleased. I need to send an unrelated PR amending the changelog, so if you have any suggestions on improvements on that, I can send it :)

Reply:
OK!